_markets/de/buy-resell/products/_archive.mdx

```diff
@@ -1,11 +1,22 @@
-### POS Archiv
+We offer two solutions for archiving data, see their Product Features below and [find more product details here](https://fiskaltrust.de/produkte/).
+MeinFiskal is described with its [technical details here](https://docs.fiskaltrust.cloud/de/docs/posdealers/buy-resell/products/3rd-party/datev-meinfiskal).
 
-The POS Archive is a **cloud service** which provides a searchable, secure, and **audit-proof** database for Middleware receipts. Activating it for a CashBox will also enable its Middleware instance to regularly push its new receipts into the archive, where they'll be securely persisted in a revision-safe storage. Due to german regulations, the data is kept for **at least ten years**.
+| Product Features | DATEV MeinFiskal    | PosArchiv           |
+|:----------------------|:--------------------|:----------------------|
+|Included in the Carefree Bundle |Yes  |Yes
+|Available as single product |No (in progress)  |Yes
+|Setup for multiple PosOperators |No |Yes  |
```

Comment:
What does that mean?

Reply:
We can use Rollout Management for multiple PosOperators, but do we want to name the process Rollout? Danaiel and me siscussed that and agreed on "Setup for". He added, that there will be no option for such a setup with DATEV

Reply:
I'd go with a different approach, since this has nothing to do with the rollout, but with contracting.

We could call this "3rd party agreement required" (DATEV: Yes, PosArchive: no)